tomb/src/utils/mod.rs

```diff
@@ -1,17 +1,17 @@
+/// Utils specific to the Bundle pipeline
+pub mod bundle;
```

Comment:
These kinds of renames can really expand the size of these PRs and make the signal-to-noise ratio pretty bad. In the future please keep large-scale renames, or reorganizations in their own PRs away from features and bug fixes.

Reply:
got it